### PR TITLE
Implement ros_time arg option for run_sensors.sh

### PR DESCRIPTION
**Pull request by GreatAlexander**
- ros_time (false by default) option sets lidars (ouster/velodyne) to
 not use GPS/PPS time synchronisation for their ROS msg timestamps
 - This is useful for testing the AV sensors when indoors, as indoors
 there is no GPS/PPS signal and thus sensors timestamps will be wrong
 - Instead ros_time uses the ROS "clock" to set the timestamps, which is
 good enough for some testing purposes

### CODE REVIEW

**Comment by GreatAlexander** [comment]
To be discussed: Perhaps it's best to instead create a fake NMEA/PPS node which we run indoors, and leave the operation of the sensor nodes to their default (outdoors) operation.

**Comment by hect95** [comment]
> To be discussed: Perhaps it's best to instead create a fake NMEA/PPS node which we run indoors, and leave the operation of the sensor nodes to their default (outdoors) operation.

I am generally  okay with this. I also can not come up with a better way to do this 😢 .

Let's resolve the conflicts 😄

File: run_sensors.sh
Comment (GreatAlexander) [inline run_sensors.sh:1]: @hect95 Plz fix this file's merge conflicts

Comment (hect95) [reply run_sensors.sh:1]: Done

**Review by hect95 — APPROVED** [review]
Thanks for your hard work!

**Review by hect95 — APPROVED** [review]

**Pull request merged**